test/config/MockEmploymentUserDataRepository.scala

```diff
@@ -43,7 +43,14 @@ trait MockEmploymentUserDataRepository extends MockFactory {
   }
   def mockUpdate(employmentUserData: EmploymentUserData, response: Boolean): CallHandler1[EmploymentUserData, Future[Boolean]] = {
```

Comment:
employmentUserData is never used, can be removed

Reply:
Updated